src/megachatapi_impl.cpp

```diff
@@ -1855,13 +1855,10 @@ void MegaChatApiImpl::sendPendingRequests()
                 }
             }
 
-            const char* publicHandle = request->getLink();
-            MegaChatHandle ph = publicHandle ? karere::Id(publicHandle, strlen(publicHandle)).val : MEGACHAT_INVALID_HANDLE;
-
             std::vector<::promise::Promise<void>> promises;
             for (unsigned int i = 0; i < handleList->size(); i++)
             {
-                promises.push_back(mClient->userAttrCache().getAttributes(handleList->get(i), ph));
+                promises.push_back(chatroom->chat().requestUserAttributes(handleList->get(i)));
```

Comment:
Maybe the request can finish with an MegaChatError if we call this method with a private chatroom. The method `requestUserAttributes` returns a promise::_Void() and doesn't do anything. For private chatrooms, it isn't usefull

Reply:
This comment is obsolete, right?